enterprise/server/util/vfs_server/vfs_server.go

```diff
@@ -1037,15 +1008,18 @@ func (p *Server) Rename(ctx context.Context, request *vfspb.RenameRequest) (*vfs
 
 	oldChildNode.mu.Lock()
 	newBackingPath := ""
+	oldBackingPath := ""
```

Comment:
This fixed an unrelated bug in rename handling. 

As mentioned in the PR description, this gnarly code will go away shortly since we no longer need to care about making sure the paths in the scratch space match 1:1 with those in the fuse mount.